Core/ConfigManager.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium.Interactions;
+using System.Configuration;
+
+namespace Core
+{
```

Comment:
Why do the indices of the products need to be kept in the config? You can declare const fields in your test base that save the relevant index numbers